src/systems/setup.rs

```diff
@@ -1,7 +1,33 @@
 use bevy::prelude::*;
+use bevy_prototype_lyon::{
+    prelude::{Fill, GeometryBuilder, ShapeBundle},
+    shapes::{Rectangle, RectangleOrigin},
+};
 
 pub fn spawn_camera(mut commands: Commands) {
-    commands.spawn(Camera2dBundle::default());
+    let scale = 1.0 / 48.0;
+    commands.spawn(Camera2dBundle {
+        transform: Transform::from_xyz(0.0, 0.0, 0.0).with_scale(Vec3 {
+            x: scale,
+            y: scale,
+            z: scale,
+        }),
+        ..default()
+    });
+}
+
+pub fn place_unit_tile(mut commands: Commands) {
+    commands.spawn((
+        ShapeBundle {
+            path: GeometryBuilder::build_as(&Rectangle {
+                extents: Vec2 { x: 1.0, y: 1.0 },
+                origin: RectangleOrigin::BottomLeft, // Y+ Up, X+ Right
+            }),
```

Comment:
@pugad  fyi, bevy's origin is bottom left (upwards is positive Y, rightwards is positive X)